src/feature/hs/hs_service.c

```diff
@@ -1136,23 +1136,31 @@ parse_authorized_client(const char *client_key_str)
                          SPLIT_SKIP_SPACE, 0);
   /* Wrong number of fields. */
   if (smartlist_len(fields) != 3) {
+    log_warn(LD_REND, "The file is in a wrong format.");
```

Comment:
Let's say which file -- it's not going to help the user otherwise.

Reply:
This has been fixed with a future commit in this branch.